docs/contributing/index.md

```diff
@@ -1,15 +1,67 @@
 # Contributing
 
-!!! warning
+This page explains how to contribute to Autoware.
 
-    Under Construction
+First, confirm the [Code of conduct](#contribution-workflows) section for the general manners in contributions.
+
+See the [Contribution workflows](#contribution-workflows) section for the workflows.  
+Also, confirm the following our guidelines depending on your situation:
+
+- [Coding guidelines](coding-guidelines/index.md): when you write source code
+- [Testing guidelines](testing-guidelines/index.md): when you write tests
+- [Documentation guidelines](documentation-guidelines/index.md): when you write documents
+- [Pull request guidelines](pull-request-guidelines/index.md): when you submit pull requests
+- [Discussion guidelines](discussion-guidelines/index.md): when you discuss with other contributors
+- [Support guidelines](../help/support-guidelines.md): when you need any help
+
+It is recommended to read the [Contributing pages in ROS 2 Docs](https://docs.ros.org/en/rolling/Contributing.html) as well because Autoware's guidelines generally depend on them.
+
+For general information about OSS contributions, [Open Source Guides](https://opensource.guide/) by GitHub is a good resource.
+
+!!!info
+
+    If you want to apply a new or different rule, you can open a new discussion thread and propose it.
```

Comment:
```suggestion
    If you want to make change to these or any other rules/guidelines, you can open a new discussion thread and propose it.
```

Reply:
`make change` -> `make a change` or `make changes`?

Reply:
How about this?

```
If you want to make a change to any guidelines in this documentation site, you can open a new discussion thread and propose it.
```

Reply:
`make changes` is better.

`to any guidelines` here the `any` is redundant.

I've updated it, does it work for you?

Reply:
Yeah! Thank you.